internal/recon/fixtures/recon_failed_collect.prom

```diff
@@ -87,7 +103,7 @@ swift_cluster_storage_used_percent_by_disk{disk="sdb14",storage_ip="10.0.0.1"} 0
 # TYPE swift_recon_task_exit_code gauge
 swift_recon_task_exit_code{query="--timeout=1 --diskusage --verbose"} 1
 swift_recon_task_exit_code{query="--timeout=1 --driveaudit --verbose"} 1
-swift_recon_task_exit_code{query="--timeout=1 --md5 --verbose"} 1
+swift_recon_task_exit_code{query="--timeout=1 --md5 --verbose"} 0
```

Comment:
Why has this exit code changed?

Reply:
I accidentally [deleted](https://github.com/sapcc/swift-health-exporter/pull/12/commits/31a45e0a69eeafff41680cd37b6d7f802a6d9f74#diff-5a2fbac2eeadf25cad0fc160eb93740c9beb163e820161d77b79aa25bf8c0729L148) an assignment statement during copy-paste. It's fixed now.